domains/transfer/TransferAgreementAutomation/Worker/Service/ProjectOriginWalletService.cs

```diff
@@ -36,9 +36,9 @@ AutomationCache cache
         this.cache = cache;
     }
 
-    public async Task TransferCertificates(TransferAgreementDto transferAgreement)
+    public async Task TransferCertificates(TransferAgreement transferAgreement)
```

Comment:
Once again, why does the concept of certificates exist in transfer subsystem?

Reply:
this is a transfer agreement?